.devcontainer/Dockerfile

```
@@ -38,9 +37,7 @@ RUN ls -la ${HOME}/ && ls -la ${CMDLINE_HOME} && echo ${CMDLINE_HOME} && chown -
 RUN chmod +r+w+x -Rv ${HOME}/Android/Sdk
 
 RUN yes | sdkmanager --licenses
-RUN yes | sdkmanager --install "build-tools;34.0.0"
-RUN yes | sdkmanager --install "platforms;android-21"
-RUN yes | sdkmanager --install "platforms;android-22"
+RUN yes | sdkmanager --install "build-tools;36.0.0"
```

AGP 8.12 uses Build Tools 35.0.0: https://developer.android.com/build/releases/gradle-plugin#compatibility
May we should use that version here instead?
```suggestion
RUN yes | sdkmanager --install "build-tools;35.0.0"
```

---

I can install them both. Actually it doesn't matter too much, we can bump build-tools for the project or install missed part automatically with gradle.